src/packages/core/src/base-entity.ts

```diff
@@ -25,7 +25,7 @@ export const AdminUISettingsMap = new Map<string, AdminUISettingsType>();
 
 @ObjectType()
 export class GraphQLEntity<D extends BaseDataEntity> {
-	public id?: string;
+	public id?: string | number;
```

Comment:
Why are we now supporting ids as numbers? Usually they're like "1"

Reply:
it depends on the data source they could be strings or numbers. We normally store as BigInt which has to be represented as a string in JS. But another DB might just use `int` which can be represented as a number.